pkg/fleet/installer/setup/djm/emr.go

```diff
@@ -52,6 +49,22 @@ var (
 			Value: "true",
 		},
 	}
+	emrLogs = []common.IntegrationConfigLogs{
+		{
+			Type:    "file",
+			Path:    hadoopLogFolder + "*/*/stdout",
+			Source:  "worker_logs",
+			Service: "hadoop-yarn",
+			Tags:    "log_source:stdout",
```

Comment:
```suggestion
			Source:  "hadoop-yarn",
			Service: "emr-logs",
```

- source should be the name of the integration: `hadoop-yarn` seems like a good value
- service is more complex since multiple service can run on the same cluster, I was thinking of a global name like `emr-logs`

The filename (stderr vs stdout) should directly be capture directly by the agent